README.md

```diff
@@ -1,8 +1,304 @@
+- [Discussing RFCs](https://github.com/amethyst/rfcs/pulls)
+- [Accepted RFCs](https://github.com/amethyst/rfcs/tree/master)
+- [RFCs Awaiting Implementation](https://github.com/amethyst/rfcs/issues?utf8=✓&q=label%3A%22RFC%3A+Accepted%22+-label%3A%22Status%3A+Working%22+)
 
-## Creating a new RFC
+## Opening
+[Opening]: #opening
 
-The specifics of the RFC process are not yet discussed, but will be documented in the first RFC.
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Amethyst community and
+the [sub-team]s.
+
+The "Request for Comments" (RFC) process is intended to provide a consistent
+and controlled path for new features to enter the engine and associated
+libraries, so that all stakeholders can be confident about the direction in which
+the engine is evolving.
+
+You may join our Discord here: https://discord.gg/dqe9UYE and the Amethyst Community Forum here: https://community.amethyst-engine.org.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+- [Opening]
+- [Table of Contents](#table-of-contents)
+- [When you need to follow this process](#when-you-need-to-follow-this-process)
+  - [Sub-team Specific Guidelines](#sub-team-specific-guidelines)
+- [Before creating an RFC](#before-creating-an-rfc)
+- [What the process is](#what-the-process-is)
+- [Tracking Issues](#tracking-issues)
+  - [Labels and Tags](#labels-tags)
+- [The RFC life-cycle](#the-rfc-life-cycle)
+  - [Reviewing RFCs](#reviewing-rfcs)
+  - [Implementing an RFC](#implementing-an-rfc)
+  - [RFC Postponement](#rfc-postponement)
+- [Help this is all too informal!](#help-this-is-all-too-informal)
+- [License](#license)
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Amethyst, its component crates, or the RFC process itself. The definition of substantial
+is somewhat fluid, and to an extent set by the community. Some examples might include:
+
+- A change to a different crate used for physics calculations
+- Changes to how the editor communicates with the engine
+- Significant protocol changes to the networking library
+- Anything that has the potential to produce a substantial negative impact on users
+
+Some changes do not require an RFC:
+
+- Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+  not change meaning".
+- Additions that strictly improve objective, numerical quality criteria
+  (warning removal, speedup, better platform coverage, more parallelism, trap
+  more errors, etc.)
+- Additions only likely to be _noticed by_ other developers-of-amethyst,
+  invisible to users-of-amethyst.
+- More tests and benchmarks
```

Comment:
I might also add "convenience functions" here for simple quality of life improvements that don't substantially alter the API, just make it easier to use.

Reply:
"that do not break backwards compatibility" I would assume should be in there?

Reply:
Yup

Reply:
Done